src/lib/Libifl/pbsD_Preempt_Jobs.c

```diff
@@ -87,7 +87,8 @@ preempt_jobs_send(int connect, char **preempt_jobs_list)
 	}
 
 	if (dis_flush(connect)) {
-		pbs_errno = PBSE_PROTOCOL;
+		if (pbs_errno == PBSE_NONE)
```

Comment:
why checking for PBSE_NONE? I could not see this check with any of the errors above.

Reply:
Does dis layer set pbs error code? I don't think so. but if that's the case you might want to add the check to other places in the function.